cassandane/Cassandane/Cyrus/Sieve.pm

```diff
@@ -4358,6 +4358,163 @@ EOF
     $self->assert_str_equals('DISPLAY', $ical->{entries}[1]{entries}[0]{properties}{action}[0]{value});
 }
 
+sub test_imip_override2
```

Comment:
could rename that to something more specific? From what I see in the test, it's something like `test_imip_preserve_alerts` ?

Reply:
Done